crt_portal/cts_forms/urls.py

```diff
@@ -23,5 +23,5 @@
     path('attachment/report/<int:report_id>/', ReportAttachmentView.as_view(), name='save-report-attachment'),
     path('attachment/<int:attachment_id>/', RemoveReportAttachmentView.as_view(), name='remove-report-attachment'),
     path('trends/', TrendView.as_view(), name='trends'),
-    path('dashboard/', dashboard_view, name='dashboard'),
+    path('dashboard/', dashboard_view, name='dashboard')
```

Comment:
@billyhunt Should I revert this change, too?  I don't want to clash with your work.

Reply:
Thanks! 🙏🏻